sycl/unittests/Extensions/VirtualFunctions/RuntimeLinking.cpp

```diff
@@ -128,10 +150,16 @@ static sycl::unittest::MockDeviceImage Imgs[] = {
     generateImage({"KernelF"}, "set-f", /* uses vf set */ true, PROGRAM_F),
     generateImage({"DummyKernel7"}, "set-f", /* provides vf set */ false,
                   PROGRAM_F0),
-    generateImage({"KernelG"}, "set-f", /* uses vf set */ true, PROGRAM_F1)};
+    generateImage({"KernelG"}, "set-f", /* uses vf set */ true, PROGRAM_F1),
+    generateImage({"KernelH"}, "set-h", /* uses vf set */ true, PROGRAM_H,
+                  false, {}),
+    generateImage({"DummyKernel7"}, "set-h", /* provides vf set */ false,
```

Comment:
Yeah, in retrospect, `provides vf set false` reads as "this does not provide a vf set", even though it does :)

It probably should have been a bool constant here instead of a comment, i.e.:

```
// Helpers for UsesVFSets argument of generateImage
constexpr bool provides_vf_set = false;
constexpr bool uses_vf_set = true;
// ...
  generateImage(..., provides_vf_set, ...),
  generateImage(..., uses_vf_set, ...),
// ...
```

Minor comment, we can (and probably should) address it in a separate PR to reduce amount of unrelated changes here